gradle.properties

```diff
@@ -1,4 +1,4 @@
-base_version=0.7.0
+base_version=0.7.1
```

Comment:
0.x breaking change? `tableName` could be optional as its current value - `events`.

Reply:
Optional here - https://github.com/cultureamp/kestrel/pull/5/files#diff-6e363faa6074f69d02aa66a6967a34b81050150d9c2ccb0945d10aa30a9b7b02R195

Reply:
Oh man I totally forgot that @admackin wanted to do this 🤦🏻

Reply:
Ah you're right I guess it's technically a breaking change since that class is instanciable from outside. I see what you mean. I like your idea to make it optional 👍🏻

Reply:
Made it optional